get_alerts.py

```diff
@@ -6,10 +6,55 @@
 import csv
 import os
 import requests
+import re
 
 API_ENDPOINT = 'https://app.threatstack.com/api/v1/alerts'
 
 TIMESTAMP_FIELDS = ['created_at', 'expires_at', 'last_updated_at']
+
+def main():
+    '''
+    Main method. Collect command line arguments, and pass them into the get_alerts method
+    '''
+    PARSER = argparse.ArgumentParser(description='Query threatstack alerts.')
+    PARSER.add_argument('--auth', help='authentication token', dest='auth', required=True)
+    PARSER.add_argument('--org', help='organization', dest='org', required=True)
+    PARSER.add_argument('--fields', help='the fields to return from the api', dest='fields',
+                        required=False, default='severity,last_updated_at,title')
+    PARSER.add_argument('--start', help='start date', dest='start',
+                        required=False,
+                        default=datetime.datetime.utcnow() - datetime.timedelta(days=1))
+    PARSER.add_argument('--end', help='end date', dest='end',
+                        required=False,
+                        default=datetime.datetime.utcnow())
+    PARSER.add_argument('--count', help='number of alerts to return', dest='count',
+                        required=False,
+                        default=20)
+    PARSER.add_argument('--outfile',
+                        help='the name (or path including csv) of the csv file to output',
+                        dest='out',
+                        required=False,
+                        default=os.getcwd()+'/'+'alerts.csv')
+    PARSER.add_argument('--omitheader', action='store_true', default=False, dest='omitheader')
+    PARSER.add_argument('--filters', help = 'filters to apply', dest = 'filters', required = False)
```

Comment:
I considered doing this, but I ultimately chose using 'and' as a separator so that we can eventually use 'or' as well